### ISSUE TITLE
Events not sent after upgrade

### ISSUE THREAD

**Issue by bluewindthings** [owntracks/ios#158]
After upgrading to 8.0.6 and reloading waypoints (see screenshot below) events are not generated / sent. Creating new waypoints does not help.

![image1](https://cloud.githubusercontent.com/assets/3198092/7313888/b668d928-ea5e-11e4-91dc-09841122ed57.PNG)

**Comment by jpmens** [comment]
This has been resolved in newer versions which are in alpha testing for us. They will be released when we're satisfied. Closing now.

**Comment by jpmens** [comment]
Let me rephrase that, please. Released when we're confident everything works well. :-)

**Comment by bluewindthings** [comment]
Ok thanks! Just wanted to be sure you didn't miss this.
